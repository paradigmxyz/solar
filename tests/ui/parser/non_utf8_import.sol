import "./auxiliary/non_utf8.sol"; //~ ERROR: stream did not contain valid UTF-8
