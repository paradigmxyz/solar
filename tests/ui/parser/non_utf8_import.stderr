error: couldn't read ROOT/tests/ui/parser/auxiliary/non_utf8.sol: stream did not contain valid UTF-8
  --> ROOT/tests/ui/parser/non_utf8_import.sol:LL:CC
   |
LL | import "./auxiliary/non_utf8.sol";
   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

